File: crates/auto-lang/test/a2c/008_method/method.expected.c
#include "method.h"

int modulus(struct Point *s) {
    return s->x * s->x + s->y * s->y;
}

int main(void) {
    struct Point p = {.x = 3, .y = 4};
    int m = modulus(&p);
    printf("%s %d\n", "Modulus:", m);
    return 0;
}

File: crates/auto-lang/test/a2j/008_method/method.expected.js
class Point {
    constructor(x, y) {
        this.x = x;
        this.y = y;
    }

    modulus() {
        return this.x * this.x + this.y * this.y;
    }
}

function main() {
    console.log("Method defined");
}

main();
